.travis.yml

```diff
@@ -8,4 +11,7 @@ branches:
 rvm:
   - 1.9.2
   - 1.9.3
```

Comment:
Let's remove all 1.9.x versions.

Reply:
I'm fine with removing all of the `1.9.x` versions, but I figured we could merge this PR and then drop support for anything below `2.0` in the next version.